pkg/fission-cli/cmd/spec/command.go

```diff
@@ -60,13 +60,22 @@ func Commands() *cobra.Command {
 		Optional: []flag.Flag{flag.SpecDir},
 	})
 
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all Fission resources in the application specification",
```

Comment:
Since the list command only lists applied resources, how about we change it to 
```
List all applied Fission resources in the application specification
```